### PR TITLE
add dependencies and rename package

### PR DESCRIPTION
**Pull request by gauthampughazhendhi**
add dependencies and rename package

### CODE REVIEW

**Review by zackt113 — APPROVED** [review]
Looks good to me!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# strapPy
+# StrapPy
```
Comment (BooleanJulien) [inline README.md:1]:
```suggestion
# Strappy
```

File: README.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ Julien Gordon, Gautham Pughazhendhi, Zack Tang, and Margot Vore.
 
 ## License
 
-`strapPy` was created by Julien Gordon, Gautham Pughazhendhi, Zack Tang, Margot Vore. It is licensed under the terms of the MIT license.
+`StrapPy` was created by Julien Gordon, Gautham Pughazhendhi, Zack Tang, Margot Vore. It is licensed under the terms of the MIT license.
```
Comment (BooleanJulien) [inline README.md:32]:
```suggestion
`Strappy` was created by Julien Gordon, Gautham Pughazhendhi, Zack Tang, Margot Vore. It is licensed under the terms of the MIT license.
```

File: README.md [hunk 3]
```diff
@@ -29,8 +29,8 @@ Julien Gordon, Gautham Pughazhendhi, Zack Tang, and Margot Vore.
...
 
 ## Credits
 
-`strapPy` was created with [`cookiecutter`](https://cookiecutter.readthedocs.io/en/latest/) and the `py-pkgs-cookiecutter` [template](https://github.com/py-pkgs/py-pkgs-cookiecutter).
+`StrapPy` was created with [`cookiecutter`](https://cookiecutter.readthedocs.io/en/latest/) and the `py-pkgs-cookiecutter` [template](https://github.com/py-pkgs/py-pkgs-cookiecutter).
```
Comment (BooleanJulien) [inline README.md:36]:
```suggestion
`Strappy` was created with [`cookiecutter`](https://cookiecutter.readthedocs.io/en/latest/) and the `py-pkgs-cookiecutter` [template](https://github.com/py-pkgs/py-pkgs-cookiecutter).
```

**Review by BooleanJulien — APPROVED** [review]
I think I caught a couple of missed cases, otherwise looks good!

File: README.md (hunk 1 above)
Comment (gauthampughazhendhi) [reply README.md:1]: @BooleanJulien that's intentional, just followed the NumPy convention for readme after checking their repo

**Pull request merged**